### PR TITLE
making some bad cookies.

### PR DESCRIPTION
**Pull request by cnishina**
- cookie expires in 1 minute
- does a check that the cookie is set
- refreshing does not allow to go back to login until it expires

### CODE REVIEW

File: e2e/prisoners.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    browser.get('/prisoners');
+    expect(browser.getCurrentUrl()).toBe(browser.baseUrl + '/assets/login.html');
+  });
+
+  it('should log in with fake credentials', () => {
+    element(by.css('.user')).sendKeys('foo');
+    element(by.css('.password')).sendKeys('bar');
+    element(by.css('button#login')).click();
+    browser.waitForAngularEnabled(true);
+    browser.sleep(300);
```
Comment (heathkit) [inline e2e/prisoners.e2e-spec.ts]: We shouldn't encourage browser.sleep - could we do something with browser.wait instead?

File: e2e/prisoners.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import {browser, by, element, ExpectedConditions as EC} from 'protractor';
+
+fdescribe('Prisoner manifest', () => {
```
Comment (heathkit) [inline e2e/prisoners.e2e-spec.ts]: drop the fdescribe

File: src/app/auth-guard.service.ts [hunk 3]
```diff
@@ -10,11 +10,13 @@ export class AuthGuard implements CanActivate {
 
   public canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot) {
     // TODO(Milestone 2): Fix the typo
-    if (route.url[0].path == "prisoner") {
+    console.log(window.document.cookie.toString());
```
Comment (heathkit) [inline src/app/auth-guard.service.ts]: extra console.log?

**Review by heathkit — APPROVED** [review]

File: e2e/prisoners.e2e-spec.ts (hunk 1 above)
Comment (cnishina) [reply e2e/prisoners.e2e-spec.ts]: done.

File: e2e/prisoners.e2e-spec.ts (hunk 2 above)
Comment (cnishina) [reply e2e/prisoners.e2e-spec.ts]: removed.

File: src/app/auth-guard.service.ts (hunk 3 above)
Comment (cnishina) [reply src/app/auth-guard.service.ts]: removed.

**Pull request merged**